src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -79,6 +80,23 @@ public void satisfiesEqualsContract() throws IOException {
             ),
             "Content-Disposition: form-data; name=\"data\"; filename=\"a.bin\""
         );
+        final String boundary = "AaB02x";
```

Comment:
@yegor256 it is testing <code>RqMultipart.Base</code> equality and base is waiting request not multipart request. That is the reason why i leave it like that.